src/editor.h

```diff
@@ -137,6 +137,7 @@ typedef struct GeanyEditorPrefs
 	gboolean	long_line_enabled;
 	gint		autocompletion_update_freq;
 	gint		scroll_lines_around_cursor;
+	gboolean	ime_interaction; /* input method editor's candidate window behaviour */
```

Comment:
This should be an int and use on of the Scintilla constants, `SC_IME_WINDOWED` or `SC_IME_INLINE`, so it can be easily extended in the future, and have a semantic meaning.